plugins/dell-kestrel/fu-dell-kestrel-hid-device.c

```diff
@@ -268,10 +268,8 @@ fu_dell_kestrel_hid_device_write_firmware_pages(FuDellKestrelHidDevice *self,
 	return TRUE;
 }
 
-static gboolean
-fu_dell_kestrel_hid_device_verify_chunk_result(FuDellKestrelHidDevice *self,
-					       guint chunk_idx,
-					       GError **error)
+static FuDellKestrelEcRespToChunk
+fu_dell_kestrel_hid_device_verify_chunk_result(FuDellKestrelHidDevice *self, GError **error)
```

Comment:
nah, this should return boolean -- if you need to return FuDellKestrelEcRespToChunk then add it as a `FuDellKestrelEcRespToChunk *resp` and treat it as a `out` parameter.

Reply:
should be fixed.